drivers/volume/portworx/schedops/k8s-schedops.go

```diff
@@ -479,6 +483,25 @@ func (k *k8sSchedOps) IsPXReadyOnNode(n node.Node) bool {
 	return true
 }
 
+// IsPXInstalled returns true/false if px label is enabled/disabled on kubernetes node
+func (k *k8sSchedOps) IsPXInstalled(n node.Node) bool {
+	isPxInstalled := true
+	kubeNode, err := k8s.Instance().GetNodeByName(n.Name)
+	if err != nil {
```

Comment:
return error if you are not able to find the node
also add a small retry before you error out; may be 30 seconds or 1 min at an interval of 5 seconds